myblog/blog/models.py

```diff
@@ -9,10 +9,23 @@ class BlogPost(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     img = models.ImageField(upload_to='uploads/%Y/%m/%d/', blank=True, null=True)
     safe = models.BooleanField(default=True)
+    tagged_users = models.ManyToManyField(User, through='UserTag', related_name='tagged_posts')
     
     @property
     def comments_count(self):
         return Comment.objects.filter(blogpost=self).count()
+    
+    @property
+    def tagged_count(self):
+        return self.tagged_users.count()
+    
+    @property
+    def last_tag_date(self):
+        last_tag = self.usertags.order_by('-created_at').first()
+        if last_tag:
+            return last_tag.created_at
+        else:
+            return None
```

Comment:
@ejariza-evowill, I Checked this, and I think I can't. With this one line, I get an instance of UserTag. But it's needed the field created_at and not the object. When I put the line as ```return self.usertags.order_by('-created_at').first().created_at or None``` to return the date, I probably get an Attribute error, because it's trying to access the created_at attribute of an object that might be None. Specifically, the order_by('-created_at').first() query might return None if there are no related UserTag objects for the blog post. If I get the 'NoneType' object, it will not have attributed 'created_at' and the code breaks

Reply:
Oh, yes, you're right here, I think that in this case you could use ternary operator, but let it be like that by now.